src/app.js

```diff
@@ -29,6 +30,7 @@ app.use(cookieParser());
 app.use('/api/resources', resourceModule);
 app.use('/api/users', userController);
 app.use('/api/auth', authController);
+app.use('/api/password-recovery', passwordController);
```

Comment:
Use here just `/api/passwords` and come up with a better naming in the controller
`password-recovery` is acceptable but it's not common
Here we need to use more common naming